releases.yml

```diff
@@ -4,6 +4,10 @@
       url: /docs/patterns/equal-heights
       status: Updated
       notes: We've introduced an option to use <a href="/docs/patterns/images#highlighted-image">highlighted images</a> within the equal heights pattern.
+    - component: Single-tier pricing block pattern variant
+      url: /docs/patterns/pricing-block#1-block
+      status: New
+      notes: We've added a new single-tier variant for the pricing block pattern.
```

Comment:
Please move to `4.29.0` as `4.28.0` has been released - also update `package.json` version number